This patch is based on the upstream commit described below, adapted for use
in the Debian package by Peter Michael Green.

commit 71c139c81e1b6c016180364281b422136051274f
Author: David Peter <mail@david-peter.de>
Date:   Wed Aug 3 13:00:00 2022 +0200

    Updates for indicatif 0.17

Index: hyperfine/src/output/progress_bar.rs
===================================================================
--- hyperfine.orig/src/output/progress_bar.rs
+++ hyperfine/src/output/progress_bar.rs
@@ -1,4 +1,5 @@
 use indicatif::{ProgressBar, ProgressStyle};
+use std::time::Duration;
 
 use crate::options::OutputStyleOption;
 
@@ -14,7 +15,8 @@ pub fn get_progress_bar(length: u64, msg
         OutputStyleOption::Basic | OutputStyleOption::Color => ProgressStyle::default_bar(),
         _ => ProgressStyle::default_spinner()
             .tick_chars(TICK_SETTINGS.0)
-            .template(" {spinner} {msg:<30} {wide_bar} ETA {eta_precise}"),
+            .template(" {spinner} {msg:<30} {wide_bar} ETA {eta_precise}")
+            .expect("no template error"),
     };
 
     let progress_bar = match option {
@@ -22,7 +24,7 @@ pub fn get_progress_bar(length: u64, msg
         _ => ProgressBar::new(length),
     };
     progress_bar.set_style(progressbar_style);
-    progress_bar.enable_steady_tick(TICK_SETTINGS.1);
+    progress_bar.enable_steady_tick(Duration::from_millis(TICK_SETTINGS.1));
     progress_bar.set_message(msg.to_owned());
 
     progress_bar
Index: hyperfine/Cargo.toml
===================================================================
--- hyperfine.orig/Cargo.toml
+++ hyperfine/Cargo.toml
@@ -48,7 +48,7 @@ version = "2.0"
 version = "1.1"
 
 [dependencies.indicatif]
-version = "0.16"
+version = "0.17"
 
 [dependencies.rand]
 version = "0.8"
